File: 프로그래머스/src/프로그래머스_02272022/부족한_금액_계산하기.cpp
using namespace std;

long long solution(int price, int money, int count)
{
    long long total = 0;
    for(int i = 1 ; i<=count ; i++)
    {
        total += price * i ;
    }
    return total - money <0 ? 0 : total - money;
}

File: 2026/Level1/부족한금액계산하기.py
# https://school.programmers.co.kr/learn/courses/30/lessons/82612?language=python3
# y = price * count
def ride(p,c):
    return p*c

def solution(price, money, count):
    answer = money
    for x in range(1,count+1) :
        answer -= ride(price, x)
    return answer*-1 if answer <=0 else 0